cdn_lambda/functions/origin_response/origin_response.py

```diff
@@ -24,6 +25,22 @@ def handler(self, event, context):
         request = event["Records"][0]["cf"]["request"]
         response = event["Records"][0]["cf"]["response"]
 
+        try:
+            want_digest = request["headers"]["want-digest"][0]["value"]
+        except KeyError:
+            LOG.debug(
+                "Could not read want-digest from response", exc_info=True
+            )
+            want_digest = None
+
+        if want_digest == "sha-256":
+            response["headers"]["digest"] = [
+                {
+                    "key": "Digest",
+                    "value": f"sha-256={basename(request['uri'])}",
```

Comment:
This isn't correct, the digest is supposed to be base64 encoded.

Reply:
I'm not sure what's meant by this. I've tried encoding the sha-256 object key but then it can't be used in the string. I would prefer to calculate the "real" checksum here but I'm not sure that's reasonable.